.clang-tidy

```diff
@@ -11,7 +31,6 @@ CheckOptions:
   - { key: readability-identifier-naming.ProtectedMemberPrefix,   value: m_ }
   - { key: readability-identifier-naming.MemberCase,              value: camelBack }
   - { key: readability-identifier-naming.ConstexprVariableCase,   value: UPPER_CASE }
-  - { key: readability-identifier-naming.EnumCase,                value: UPPER_CASE }
```

Comment:
Why did you remove this?
We actually have this coding rule, did it produce somewhere problems?

Reply:
@elfenpiff this is duplicated `readability-identifier-naming.EnumCase`, above we have 

```
{ key: readability-identifier-naming.EnumCase,                value: CamelCase }
```

Most of the enum classes I saw use camel case

Reply:
@elfenpiff or you think we need to switch enum classes to UPPER_CASE?

Reply:
@elfenpiff @r7vme I think we use mostly `UPPER_CASE` for `EnumConstantCase`. A notable exception to this is the `Error` enum which uses a `k` prefix for its values.

Reply:
ack, will remove rule above with camel case then

Reply:
@r7vme the rule with camel case is valid, but for `EnumCase` not `EnumConstantCase`

Reply:
done